### DISCUSSION TITLE
teslamate(v1.30.1) - korea / The status of the vehicle is displayed as unavailable.

### DISCUSSION THREAD

**Discussion by Gimgyu** [teslamate-org/teslamate#4194] [Q&A]
teslamate - korea / The status of the vehicle is displayed as unavailable.
![캡처](https://github.com/user-attachments/assets/93197d57-5121-4bee-9d5e-e751109cdf57)

teslamate:1.30.1
tesla : model y 2024.20.7

**Comment by nU21** [comment]
I'm getting the same error.

**Comment by vincep5** [comment]
Same error for me today too. I restarted the Teslamate host and still the same thing. The Tesla App is showing the car fine

**Comment by Nakatomi2010** [comment]
I'm getting the same error on _one_ of my cars, but not the other one.

My 2019 Model 3 SR+ (2024.26.20) has stopped phoning home as of last night, but my 2022 Model Y Performance (2024.15.15) is still phoning home.

That said, the Model 3 SR+ is still talking to my Home Assistant instance.

So, I'm not sure why it's car and TeslaMate specific.

I've restarted TeslaMate, and the car, same thing.

**Comment by sdit88** [comment]
Also having this issue, two cars showing as unavailable suddenly this afternoon.
Third car still working fine.

**Reply by Nakatomi2010** [reply]
What are your vehicle firmware versions?

**Reply by sdit88** [reply]
2024.26.8 on the two that aren’t showing (2023 M3 LR AWD and 2024 MY SR), 2024.32.3 (2021 M3 SR+) on the one that is.

**Comment by jandrisani** [comment]
Same issue here, 2022 MYLR. USA East Coast. Tried restarting docker container, generating new access tokens, rebooting car. Car shows up fine in the Tesla App but Status:unavailable in Teslamate. 

Car is on 2024.32.3

**Reply by jandrisani** [reply]
Tesla API issues maybe?
```
2024-09-13 13:30:34.757 [warning] TeslaApi.Error / :too_many_request 37766
2024-09-13 13:30:34.757 car_id=1 [error] Too many request / Retry after 37766 seconds
```

**Reply by Nakatomi2010** [reply]
I just opened an issue about this here: https://github.com/teslamate-org/teslamate/issues/4195

That's the same error I'm seeing.  One of my cars has too many requests, while the other one is fine.

The car still phones home to Home Assistant fine via Tesla's fleet API, so it appears to be how TeslaMate is pulling data from Tesla that's "too many requests"

**Comment by Nakatomi2010** [comment]
For folks coming in over here.  A solution/workaround was found.

If you register with MyTeslaMate.com you can tie that site into your Tesla account and leverage the Fleet API that way to get things back online.

Not sure if this is going to be the "correct" way, or just a work around for now, but FYI.

Better solution can be found in the issue linked in my prior comment above.

**Reply by vincep5** [reply]
I did this work around yesterday using the Fleet API with myteslamate and it worked fine to reconnect my vehicle. That was until I had a few drives.  It returned data for just a few points of a 20 minute drive. It also missed completely 2 drives.  Is that what is expected with the Fleet API?
I returned back my Docker config to back what it was before and it's working like old times.

**Comment by cwanja** [comment]
Check your installations. Users that **made no changes** are reporting services returned to normal. https://github.com/teslamate-org/teslamate/issues/4195#issuecomment-2350739719
